lib/helpers/merge-exports.js

```diff
@@ -0,0 +1,12 @@
+"use strict"
+
+function mergeModules (target, module) {
+  var hasOwn = typeof Object.hasOwn === undefined? Function.call.bind(Object.prototype.hasOwnProperty) : Object.hasOwn
```

Comment:
```suggestion
  var hasOwn = typeof Object.hasOwn === 'undefined' ? Function.call.bind(Object.prototype.hasOwnProperty) : Object.hasOwn
```
also, move it up to module level